pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -2266,57 +2227,6 @@ def __setitem__(self, key, value):
         # For ASE compatibility
         self.numbers = self.get_atomic_numbers()
 
-    def __imul__(self, vec):
```

Comment:
I'm assuming this is a removal of internal multiplication method? Just removal because it wasn't used or it was used in sparselist?

Reply:
It's actually already defined in ASE. We could have shortened it much earlier, but for `SparseList` we would have needed it anyway. Now it fully uses ASE `__imul__`.